### DISCUSSION TITLE
what to put in email pass ?

### DISCUSSION THREAD

**Discussion by DHRUVPATEL369** [VaibhavArora314/StyleShare#182] [Q&A] [answered]
In .env.example i want trying to verify but it is giving me error as some password error.
so, what should be the password is it gmail password? # EMAIL_PASS=16char_app_password ?

**Comment by MeetDOD** [comment, marked as answer]
No, it will not work on your gmail password. you have to go in your google acount and search **app passwords** and follow this video for more help: https://youtu.be/u-_Ygo2wcrs?si=Zm4xIT1Ye3EZlRvd

**Reply by DHRUVPATEL369** [reply]
Ohh, i thought that it will be the gmail password, never mind and thanks
